cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,28 @@
+import { generateUser } from '../support/generate';
+
+Cypress.on('uncaught:exception', () => {
```

Comment:
It's recommended to handle uncaught exceptions in a global configuration file (like commands.js or index.js in the support folder) instead of in individual test files. This improves code maintainability and readability.